### PR TITLE
[fuzzing] Enable instantiation

### PR DESCRIPTION
**Pull request by lum1n0us**
- use a large seed to create larger binaries.
- enable instantiation.

### CODE REVIEW

File: tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc [hunk 1]
```diff
@@ -13,31 +13,41 @@
...
+        wasm_runtime_load((uint8_t *)myData.data(), Size, error_buf, 120);
+    if (!module) {
+        std::cout << "[LOADING] " << error_buf << std::endl;
+        wasm_runtime_destroy();
+        /* return SUCCESS because the failure has been handled */
+        return 0;
+    }
+
+    wasm_module_inst_t inst = wasm_runtime_instantiate(
+        module, 8 * 1024 * 1024, 16 * 1024 * 1024, error_buf, 120);
```
Comment (wenyongh) [inline tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc:37]: The input heap size and stack size are fixed, maybe we can also let the fuzz tool randomly generate them in the future. Now it looks good.

**Review by wenyongh — APPROVED** [review]

File: tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc [hunk 2]
```diff
@@ -13,31 +13,41 @@
...
+        std::cout << "[INSTANTIATE] " << error_buf << std::endl;
         wasm_runtime_unload(module);
+        wasm_runtime_destroy();
+        /* return SUCCESS because the failure has been handled */
+        return 0;
     }
-    /* destroy runtime environment */
-    wasm_runtime_destroy();
 
+    std::cout << "PASS" << std::endl;
```
Comment (xujuntwt95329) [inline tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc:46]: Should we also execute the instance? Does wasm-mutator tell us which funcs are exported? Or does it export a fixed function to be invoked?

Comment (lum1n0us) [reply tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc:46]: - Use `wasm-tools smith` to create ~10 WebAssembly modules for each set of features, including both valid and invalid ones.
- Employ these as the seed corpus.
- Allow libfuzzer to perform its function.

Comment (xujuntwt95329) [reply tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc:46]: Got it. We can firstly run this to check the coverage and further optimize the pipeline according to the results

**Review by xujuntwt95329 — APPROVED** [review]
LGTM

**Pull request merged**